rotary/config.yaml

```diff
@@ -0,0 +1,118 @@
+quality_trim_score_cutoff: 20
+# Minimum average quality score required for a read after quality trimming (disabled if quality_trim_direction is "f")
+minimum_average_quality_score_post_trim: 20
+# Minimum read length after quality trimming (otherwise is discarded)
+minimum_read_length_quality_trim: 10
+
+## Contaminant filtration
+# K-mer length for filtering contaminants
+contamination_filter_kmer_length: 31
+# NCBI genome assembly accessions to use for filtering contaminants from short read data.
+#   Default accessions: GCF_000819615.1 (PhiX), GCF_000001405.40 (human)
+#   NOTE that contaminant filtration for the human genome takes ~60 GB RAM.
+#   If you do not want to use any NCBI genome accessions for filtering, then set this variable to []
+contamination_references_ncbi_accessions: ["GCF_000819615.1", "GCF_000001405.40"]
+# Filepaths to FastA nucleotide files to use for filtering contaminants from short read data.
+#   If you do not want to use any custom FastA files for filtering, then set this variable to []
+contamination_references_custom_filepaths: []
+
+## Assembly
+# nano-hq (if <5% error) or nano-raw
+flye_input_mode: "nano-hq"
+# Optional: specify the error rate of the input reads as a fraction (e.g., use 0.03 for 3% error). This is mostly useful if you are using nano-hq with reads having >95% accuracy.
+flye_read_error: "auto"
+# Run Flye in --meta mode? Set to "True", with quotation marks, to use meta mode. Anything else (e.g., "False") will not use meta mode.
+flye_meta_mode: "False"
+# By default, Flye performs one round of long read polishing after assembly
+flye_polishing_rounds: 1
+
+## End repair
+# Flye will be used to re-assemble a short region (50-100 kb) around the ends of circular contigs.
+#   The re-assembly will be stitched to the original contigs using the circlator merge module.
+#   This process repairs an added/missing bases around the contig ends.
+# Minimum % ID needed between original and stitch contigs for merging
+circlator_merge_min_id: 99
+# Minimum hit length (bp) needed between original and stitch contigs for merging
+circlator_merge_min_length: 10000
+# Minimum distance (bp) between end of original contig and nucmer hit
+circlator_merge_ref_end: 100
+# Minimum distance (bp) between end of merge contig and nucmer hit
+circlator_merge_reassemble_end: 100
+# By default, end repair fails if a circular contig can't be end-repaired (i.e., if a contig can't be built that spans
+#   the ends of the contig). Failing to build the end-spanning contig can happen for several reasons (e.g., if that
+#   region is a repeat-rich region), even if the contig is really circular. Set keep_unrepaired_contigs to 'True' to keep
+#   the pipeline going even if end repair fails on a contig. If set to 'True', then the original assembled versions of
+#   any contigs that fail end repair will be used for downstream steps and will still be treated as circular.
+keep_unrepaired_contigs: 'False'
```

Comment:
As discussed in the general comments for this PR: change to 'True'?

Reply:
@jmtsuji Addressed in https://github.com/rotary-genomics/rotary/pull/102